### DISCUSSION TITLE
PHP Performance on kernels above 5.10.0-16-amd64 on Debian 11

### DISCUSSION THREAD

**Discussion by tvvpm** [oerdnj/deb.sury.org#1896] [Q&A]
I have php7 working over Nginx on Debian 11. It is working correctly. But in machines with kernels above 5.10.0-16-amd64 (I have test it from 5.10.0-17-amd64 to 5.10.0-20-amd64) the performance is worse. Load averages are around 100% more on these machines doing the same job.
Anyone has notice this behaviour?

**Comment by tvvpm** [comment]
![load1-day](https://user-images.githubusercontent.com/17924449/211501395-8a5bba72-90ea-42e0-b037-5ee7f85c08a5.png)
Here you can watch a load graphic in which you can see the load of web02 running 5.10.0-20-amd64 kernel, where the other machines are running 5.10.0-16-amd64

**Reply by oerdnj** [reply]
You should probably fill in a bug in the Debian bug tracker; this seems like a regression to me.

**Reply by tvvpm** [reply]
OK. For let you know ... The performance drop is due introdution of mitigations for RETbleed. Solved adding kernel boot parameter spectre_v2=off

https://lore.kernel.org/all/PH0PR05MB8448A203A909959FAC754B7AAF439@PH0PR05MB8448.namprd05.prod.outlook.com/
